build.sbt

```diff
@@ -46,15 +55,24 @@ val scalajsDomV = "2.8.0"
 def scodecV(scalaV: String) = if (scalaV.startsWith("2.")) "1.11.10" else "2.2.2"
 
 lazy val root =
-  tlCrossRootProject.aggregate(
-    core,
-    laikaIO,
-    jsInterop,
-    web,
-    searchdocsCore,
-    searchdocsIO,
-    searchdocsWeb,
-  )
+  tlCrossRootProject
+    .aggregate(
+      core,
+      laikaIO,
+      jsInterop,
+      web,
+      searchdocsCore,
+      searchdocsIO,
+      searchdocsWeb,
+    )
+    .configureRoot { root =>
+      root
+        .aggregate(plugin) // don't include the plugin in rootJVM, only in root
+        .settings(
+          crossScalaVersions := Nil,
+          scalaVersion := Scala212,
+        )
```

Comment:
I don't think these are needed anymore, they were in Laika before I upstreamed them to sbt-tl.